src/quo/foundations/common.cljs

```diff
@@ -3,8 +3,3 @@
 (def currency-label
   {:eur "€"
    :usd "$"})
-
-(def token-label
-  {:eth "Ethereum"
-   :snt "Status"
-   :dai "Dai"})
```

Comment:
Token label was hardcoded in this map. We should instead read the name from the data we fetch.